www/Component.js

```diff
@@ -19,6 +19,9 @@ sap.ui.define([
 		init: function() {
 			// call the base component's init function
 			UIComponent.prototype.init.apply(this, arguments);
+			
+			var newModel = new sap.ui.model.json.JSONModel("./model/Operators.json");
+			this.setModel(newModel, "operators");
```

Comment:
Try to write constants in a constants file in Util. Disregard this as a correction, i will create a new branch for refactoring this issues.

Reply:
Also, use this.getView().setModel. The implemented sentence sets model for the entire app. Either this or set the model on Component.js